src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+
+.nav {
+
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__link {
+    display: flex;
+    position: relative;
+    align-items: center;
+    margin-left: 20px;
+    padding: 23px 0;
+
+    line-height: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: 500;
+    color: $c-text-main;
+    transition-duration: 0.2s;
```

Comment:
You should add transition property! if you don't, the browser will watch to all properties that can change, its not good for performance

Reply:
yeah you already told me, okay I'll try to be more attentive next time!
thank you for the review!!! 🤝